starter-code/main.js

```diff
@@ -0,0 +1,229 @@
+						    rl.question(`\nType enter to continue: `, (answer) => { mainMenu() });
+						  } else {
+						    console.log(result);
+						    rl.question(`\nType enter to continue: `, (answer) => { mainMenu() });
+						  }
+						})
+							break;
+					case "8":
+					db.collection('companies').find({"name": "Facebook"}).toArray((error, result) => {
+					  if (error) {
+					    console.log(error);
+					    rl.question(`\nType enter to continue: `, (answer) => { mainMenu() });
+					  } else {
+					    console.log(result);
+					    rl.question(`\nType enter to continue: `, (answer) => { mainMenu() });
+					  }
+					})
+						break;
+					case "9":
+					db.collection('companies').find({"name": "Facebook"}, {number_of_employees: 1, _id: 0}).toArray((error, result) => {
+					  if (error) {
+					    console.log(error);
+					    rl.question(`\nType enter to continue: `, (answer) => { mainMenu() });
+					  } else {
+					    console.log(result);
+					    rl.question(`\nType enter to continue: `, (answer) => { mainMenu() });
+					  }
+					})
+						break;
+					case "10":
+					db.collection('companies').find({"name": "Facebook"}, {products: 1, _id: 0}).toArray((error, result) => {
+					  if (error) {
+					    console.log(error);
+					    rl.question(`\nType enter to continue: `, (answer) => { mainMenu() });
+					  } else {
+					    console.log((result[0].products).map((item) => item.name));
+					    rl.question(`\nType enter to continue: `, (answer) => { mainMenu() });
+					  }
+					})
+						break;
+					case "11":
+              db.collection("companies").find({"name": "Facebook"}, {relationships: 1, _id: 0}).toArray((error, result) => {
+              if (error) {
+                  console.log(error);
+                  rl.question(`\nType enter to continue: `, (answer) => { mainMenu() });
+              } else {
+								let employee = (result[0].relationships).filter((item) => item.is_past == false);
+                  console.log(employee.map((item) => item.person.first_name + " " + item.person.last_name));
+                  rl.question(`\nType enter to continue: `, (answer) => { mainMenu() });
+                  }
+              });
+              break;
+					case "12":
+              db.collection("companies").find({"name": "Facebook"}, {relationships: 1, _id: 0}).toArray((error, result) => {
+              if (error) {
+                  console.log(error);
+                  rl.question(`\nType enter to continue: `, (answer) => { mainMenu() });
+              } else {
+								let employee = (result[0].relationships).filter((item) => item.is_past);
+                  console.log(employee.length);
+                  rl.question(`\nType enter to continue: `, (answer) => { mainMenu() });
+                  }
+              });
+              break;
+						case "13":
+	              db.collection("companies").find({"relationships.person.permalink": "david-ebersman"}, {name: 1, _id: 0}).toArray((error, result) => {
```

Comment:
👍 you could have looked also for the relatioships:
**db.collection('companies').find({"relationships.person.permalink": "david-ebersman"}, {"name": 1 ,"relationships": 1}).toArray((error, result)=>{**